components/Costco.js

```diff
@@ -16,8 +16,24 @@ const Content = ({ zip }) => {
   if (!data) return <div>Loading...</div>
 
   const { date, text, screenshot, hasSlot } = data
+  let className =
+    'flex-shrink-0 flex justify-center px-2 py-1 text-sm text-white border-4 rounded disabled:cursor-not-allowed focus:shadow-outline focus:outline-none hover:bg-red-700 hover:border-red-700 disabled:opacity-75 '
```

Comment:
The `disabled:cursor-not-allowed` and `disabled:opacity-75 ` wouldn't have any effect since we never set the `<a>` be disabled.

Reply:
deleted